libs/knowledge-store/notebooks/support.ipynb

```diff
@@ -0,0 +1,596 @@
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/administration/manage-database-ip-access-list.html\n",
+      "Traversing:  https://docs.datastax.com/en/cql/astra/developing/indexing/sai/sai-overview.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/api-reference/devops-api.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/database-overview.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/cql/develop-with-cql.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/administration/manage-application-tokens.html\n",
+      "Traversing:  https://github.com/jbellis/jvector\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/load-data.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/connection-methods-comparison.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/get-started/astra-db-introduction.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/administration/manage-database-access.html\n",
+      "Traversing:  https://docs.datastax.com/en/glossary/index.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/vector-search.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/manage-collections.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/integrations/integrations-overview.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/databases/change-data-capture.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/get-started/quickstart.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/administration/configure-sso.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/api-reference/overview.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/cli-reference/astra-cli.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/tutorials/chatbot.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/integrations/embedding-providers/nvidia-nemo.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/integrations/embedding-providers/openai.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/integrations/embedding-providers/azure-openai.html\n",
+      "Traversing:  https://docs.datastax.com/en/astra-db-serverless/get-started/examples.html\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Set the question and see what documents each technique retrieves.\n",
+    "QUESTION=\"What vector indexing algorithms does Astra use?\"\n",
```

Comment:
Could we ask a question that is not answered correctly with `depth=0` and is correctly answered with a higher depth ?

Reply:
This question is not answered correctly with depth 0 and is correctly answered with higher depth, which is why this is the question that is used.

Reply:
Maybe you could emphasize that with an analysis of the results ?